bootstrap.sh

```diff
@@ -1,16 +1,25 @@
 #!/usr/bin/env bash
 
-echo "installing python"
+
+echo "installing python and postgres"
 apt update
```

Comment:
I suggest also upgrading the VM when spinning up the environment.
apt update && sudo apt upgrade -y

Reply:
Sorry, that is bad advice - it would make your development environment not reproducible. Usually you should control versions tightly and not allow "blanket upgrades" like this.

Reply:
good point. thanks